README.md

```diff
@@ -62,6 +62,11 @@ meson compile -C builddir
     Use this only when there is no other way to fix the warning.
   - `skip_dependency_check`: (Optional) List of platform specific dependency
     names that are not always provided by the project.
+  - `test_options`: (Optional) List of arguments that will be passed to
+    `meson test` command (e.g. `--timeout-multiplier`, `--no-suite`).
+  - `skip_tests`: (Optional) If set to `true` tests will not be run. This is
```

Comment:
Are we actually using skip_tests?

Reply:
I cherry-picked that commit from https://github.com/mesonbuild/wrapdb/pull/1045 where we need it.

Reply:
Note we could just merge that PR first, it is also blocked by clang-cl CI not passing, and it's fixed upstream in that case, just waiting for a release.